### PR TITLE
Add AdX passback shim

### PR DESCRIPTION
**Pull request by dwick**
👓 @tsegaran 

per http://help.adzerk.com/hc/en-us/articles/203360135-Alternate-Passback-Code-for-Use-With-Iframes

config: https://github.com/reddit/reddit-private/pull/1384

### CODE REVIEW

**Comment by dwick** [comment]
👓 @zeantsoi

File: reddit_adzerk/templates/passback.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from r2.lib.filters import scriptsafe_dumps
```
Comment (zeantsoi) [inline reddit_adzerk/templates/passback.html]: Doesn't this need to be wrapped in `<%!` `%>`?

File: reddit_adzerk/__init__.py [hunk 2]
```diff
@@ -76,6 +77,7 @@ def add_routes(self, mc):
         mc('/api/request_promo/', controller='adzerkapi', action='request_promo')
         mc('/ads/display/300x250/', controller='adserving', action='ad_300_250')
         mc('/ads/display/300x250-companion/', controller='adserving', action='ad_300_250_companion')
+        mc('/adx-passback', controller='adx', action='passback')
```
Comment (zeantsoi) [inline reddit_adzerk/__init__.py]: Not strongly opinionated, but should this be in the `/ads/` subdirectory namespace?

**Comment by zeantsoi** [comment]
:nail_care: is this something we can test on staging?

**Comment by dwick** [comment]
💇 yeah, i'll push the config and then put it up on staging.

**Comment by zeantsoi** [comment]
:fish:

**Pull request merged**